src/Components/Referrer/referrerHook.ts

```diff
@@ -31,20 +32,23 @@ const referrerData: ReferrerData = {
     jeffcoHSCM: JHSAMFBLogo,
     villageExchange: VELogo,
     cch: CCHMFBLogo,
+    lgs: LGSLogo,
   },
   logoAlt: {
     default: 'MyFriendBen home page button',
     bia: 'Benefits in Action and MyFriendBen home page button',
     jeffcoHS: 'Jeffco Human Services and MyFriendBen home page button',
     jeffcoHSCM: 'Jeffco Human Services and MyFriendBen home page button',
     cch: 'Colorado Coalition for the Homeless and MyFriendBen home page button',
+    lgs: "Let's Get Set home page button",
   },
   logoClass: {
     default: 'logo',
     jeffcoHS: 'big-logo logo',
     jeffcoHSCM: 'big-logo logo',
     villageExchange: 'big-logo logo',
     cch: 'big-logo logo',
+    lgs: 'big-logo logo',
```

Comment:
We removed this `big-logo` class in my PR today. What do you think about removing it from yours as well?